docs/get-started.md

```diff
@@ -10,16 +10,111 @@ You’ll then be ready to learn how to build your own features. Our tutorial for
+
+<%(/Expandable)%>
+
+<%(#Expandable title="Setup for Mac")%>
+
+**Step 1.** Install [**Unity 2018.2.8**](https://unity3d.com/get-unity/download/archive)
+
+- Make sure to download the **Installer** version, and select the following components during installation:
+    - **Linux Build Support**
+    - **Windows Build Support**
+
+**Step 2.** Install [**.NET Core SDK (x64)**](https://www.microsoft.com/net/download/dotnet-core/2.1)
+
+- Verified with versions `2.1.3xx` and `2.1.4xx`
+
+> **Note:** After installing the .NET Core SDK, you should restart any Unity and Unity Hub processes. This will prevent errors where Unity cannot find the `dotnet` executable.
+
+**Step 3.** Run the **<a href="https://console.improbable.io/installer/download/stable/latest/mac" data-track-link="SpatialOS Installer Downloaded|product=Docs|platform=Mac|label=Mac" target="_blank">SpatialOS installer</a>**
+
+- This installs the [SpatialOS CLI]({{urlRoot}}/content/glossary#spatial-command-line-tool-cli) , the [SpatialOS Launcher]({{urlRoot}}/content/glossary#launcher), and 32-bit and 64-bit Visual C++ Redistributables
+
+**Step 4.** Install a **code editor** if you don't have one already
+
+- We recommend either [Visual Studio](https://www.visualstudio.com/downloads/) or [Rider](https://www.jetbrains.com/rider/).
+
+**Using Visual Studio?**
+
+Once you have installed [Visual Studio](https://www.visualstudio.com/downloads/), within the Visual Studio Installer, select **.NET Core + ASP .NET Core**.
+
+**Using Rider?**
+
+Once you have installed [Rider](https://www.jetbrains.com/rider/), install the [**Unity Support** plugin](https://github.com/JetBrains/resharper-unity) for a better experience.
+
+<%(/Expandable)%>
+
+If you need help using the GDK, please come and talk to us about the software and the documentation via:
+
+  * **The SpatialOS forums** - Visit the [support section](https://forums.improbable.io/new-topic?category=Support&tags=unity-gdk) in our forums and use the unity-gdk tag.
+  * **Discord** - Find us in the [#unity channel](https://discord.gg/SCZTCYm). You may need to grab Discord [here](https://discordapp.com/).
+  * **Github issues** - Create an [issue](https://github.com/spatialos/gdk-for-unity/issues) in this repository.
+
+### Clone the SpatialOS GDK for Unity FPS Starter Project repository
+
+Clone the FPS starter project using one of the following commands: 
+
+|     |     |
+| --- | --- |
+| SSH | `git clone git@github.com:spatialos/gdk-for-unity-fps-starter-project.git` |
+| HTTPS | `git clone https://github.com/spatialos/gdk-for-unity-fps-starter-project.git` |
+
+
+### Setup dependencies by either:
+
+  * Running the bundled scripts in the `gdk-for-unity-fps-starter-project` repository: 
+      * If you are using Windows: `powershell scripts/powershell/setup.ps1`
+      *  If you are using Mac: `bash scripts/shell/setup.sh`
+  * Manually by following the instructions below.
```

Comment:
Manually by following the instructions below.

->

Following the instructions below to set up manually.